geometry/Overview.bs

```diff
@@ -1713,21 +1720,167 @@ user agents.
 
 <h2 id="cloning">Cloning</h2>
 
-<p>When a user agent is asked to <b>clone</b> an {{DOMPointReadOnly}}, {{DOMPoint}},
-{{DOMRectReadOnly}}, {{DOMRect}}, {{DOMQuad}}, {{DOMMatrixReadOnly}} or {{DOMMatrix}} object
-<var>old</var>, it must run the following steps, which return a new object of same type. These steps
-must be run atomically.
+<p>{{DOMPointReadOnly}}, {{DOMPoint}}, {{DOMRectReadOnly}}, {{DOMRect}}, {{DOMQuad}},
+{{DOMMatrixReadOnly}}, and {{DOMMatrix}} objects are <a>serializable objects</a>. [[!HTML]]
+
+<p>The <a>serialization steps</a> for {{DOMPointReadOnly}} and {{DOMPoint}}, given |value| and
+|serialized|, are:
+
+<ol>
+ <li><p>Set |serialized|.\[[X]] to |value|'s <a for=point>x coordinate</a>.
+
+ <li><p>Set |serialized|.\[[Y]] to |value|'s <a for=point>y coordinate</a>.
+
+ <li><p>Set |serialized|.\[[Z]] to |value|'s <a for=point>z coordinate</a>.
+
+ <li><p>Set |serialized|.\[[W]] to |value|'s <a for=point>w perspective</a>.
+</ol>
+
+<p>Their <a>deserialization steps</a>, given |serialized| and |value|, are:
+
+<ol>
+ <li>Set |value|'s <a for=point>x coordinate</a> to |serialized|.\[[X]].
+
+ <li>Set |value|'s <a for=point>y coordinate</a> to |serialized|.\[[Y]].
+
+ <li>Set |value|'s <a for=point>z coordinate</a> to |serialized|.\[[Z]].
+
+ <li>Set |value|'s <a for=point>w perspective</a> to |serialized|.\[[W]].
+</ol>
+
+<p>The <a>serialization steps</a> for {{DOMRectReadOnly}} and {{DOMRect}}, given |value| and
+|serialized|, are:
```

Comment:
Filed https://github.com/w3c/fxtf-drafts/issues/150 to follow up on this, thanks!